frontend/guardian.rb

```diff
@@ -80,9 +72,18 @@ def rejected_beep(gpio_command)
 DoorAuthorization.auto_upgrade!
 TagLog.auto_upgrade!
 
-close_door(gpio_command)
+close_door(gpio_command, :front_door)
 `#{gpio_command} -g mode 9 out`
-close_door(gpio_command)
+close_door(gpio_command, :front_door)
+
+close_door(gpio_command, :rollup_door)
```

Comment:
I think @raindrift said that the pin needs to be set in the closed position before it's set as an out pin.